android/build.gradle

```diff
@@ -15,6 +25,10 @@ android {
     }
 }
 
+repositories {
```

Comment:
Isn't this a duplicate to the `repositories` field in `buildscript`? (Same goes with `dependencies`)